### ISSUE TITLE
[frame_object_diff] misleading indentation (Trac #1660)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1652]
This one actually leads to wrong behavior. Yay compiler checks.

```text
/home/dschultz/Documents/combo/trunk/src/frame_object_diff/private/frame_object_diff/calibration/I3DOMCalibrationDiff.cxx:27:3: warning: this ‘else’ clause does not guard... [-Wmisleading-indentation]
   else
   ^~~~
/home/dschultz/Documents/combo/trunk/src/frame_object_diff/private/frame_object_diff/calibration/I3DOMCalibrationDiff.cxx:29:5: note: ...this statement, but the latter is misleadingly indented as if it is guarded by the ‘else’
     droopTimeConstants_[1] = cal.droopTimeConstants_[1];
     ^~~~~~~~~~~~~~~~~~~
/home/dschultz/Documents/combo/trunk/src/frame_object_diff/private/frame_object_diff/calibration/I3DOMCalibrationDiff.cxx:63:3: warning: this ‘else’ clause does not guard... [-Wmisleading-indentation]
   else
   ^~~~
/home/dschultz/Documents/combo/trunk/src/frame_object_diff/private/frame_object_diff/calibration/I3DOMCalibrationDiff.cxx:65:5: note: ...this statement, but the latter is misleadingly indented as if it is guarded by the ‘else’
     ampGains_[1] = cal.ampGains_[1];
     ^~~~~~~~~
/home/dschultz/Documents/combo/trunk/src/frame_object_diff/private/frame_object_diff/calibration/I3DOMCalibrationDiff.cxx:70:3: warning: this ‘else’ clause does not guard... [-Wmisleading-indentation]
   else
   ^~~~
/home/dschultz/Documents/combo/trunk/src/frame_object_diff/private/frame_object_diff/calibration/I3DOMCalibrationDiff.cxx:72:5: note: ...this statement, but the latter is misleadingly indented as if it is guarded by the ‘else’
     atwdFreq_[1] = cal.atwdFreq_[1];
     ^~~~~~~~~
/home/dschultz/Documents/combo/trunk/src/frame_object_diff/private/frame_object_diff/calibration/I3DOMCalibrationDiff.cxx:106:3: warning: this ‘else’ clause does not guard... [-Wmisleading-indentation]
   else
   ^~~~
/home/dschultz/Documents/combo/trunk/src/frame_object_diff/private/frame_object_diff/calibration/I3DOMCalibrationDiff.cxx:108:5: note: ...this statement, but the latter is misleadingly indented as if it is guarded by the ‘else’
     atwdDeltaT_[1] = cal.atwdDeltaT_[1];
     ^~~~~~~~~~~
```

Migrated from https://code.icecube.wisc.edu/ticket/1660
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:57", 
    "description": "This one actually leads to wrong behavior. Yay compiler checks.\n\n{{{\n/home/dschultz/Documents/combo/trunk/src/frame_object_diff/private/frame_object_diff/calibration/I3DOMCalibrationDiff.cxx:27:3: warning: this \u2018else\u2019 clause does not guard... [-Wmisleading-indentation]\n   else\n   ^~~~\n/home/dschultz/Documents/combo/trunk/src/frame_object_diff/private/frame_object_diff/calibration/I3DOMCalibrationDiff.cxx:29:5: note: ...this statement, but the latter is misleadingly indented as if it is guarded by the \u2018else\u2019\n     droopTimeConstants_[1] = cal.droopTimeConstants_[1];\n     ^~~~~~~~~~~~~~~~~~~\n/home/dschultz/Documents/combo/trunk/src/frame_object_diff/private/frame_object_diff/calibration/I3DOMCalibrationDiff.cxx:63:3: warning: this \u2018else\u2019 clause does not guard... [-Wmisleading-indentation]\n   else\n   ^~~~\n/home/dschultz/Documents/combo/trunk/src/frame_object_diff/private/frame_object_diff/calibration/I3DOMCalibrationDiff.cxx:65:5: note: ...this statement, but the latter is misleadingly indented as if it is guarded by the \u2018else\u2019\n     ampGains_[1] = cal.ampGains_[1];\n     ^~~~~~~~~\n/home/dschultz/Documents/combo/trunk/src/frame_object_diff/private/frame_object_diff/calibration/I3DOMCalibrationDiff.cxx:70:3: warning: this \u2018else\u2019 clause does not guard... [-Wmisleading-indentation]\n   else\n   ^~~~\n/home/dschultz/Documents/combo/trunk/src/frame_object_diff/private/frame_object_diff/calibration/I3DOMCalibrationDiff.cxx:72:5: note: ...this statement, but the latter is misleadingly indented as if it is guarded by the \u2018else\u2019\n     atwdFreq_[1] = cal.atwdFreq_[1];\n     ^~~~~~~~~\n/home/dschultz/Documents/combo/trunk/src/frame_object_diff/private/frame_object_diff/calibration/I3DOMCalibrationDiff.cxx:106:3: warning: this \u2018else\u2019 clause does not guard... [-Wmisleading-indentation]\n   else\n   ^~~~\n/home/dschultz/Documents/combo/trunk/src/frame_object_diff/private/frame_object_diff/calibration/I3DOMCalibrationDiff.cxx:108:5: note: ...this statement, but the latter is misleadingly indented as if it is guarded by the \u2018else\u2019\n     atwdDeltaT_[1] = cal.atwdDeltaT_[1];\n     ^~~~~~~~~~~\n}}}", 
    "reporter": "david.schultz", 
    "cc": "claudio.kopper, blaufuss", 
    "resolution": "fixed", 
    "_ts": "1550067117911749", 
    "component": "combo reconstruction", 
    "summary": "[frame_object_diff] misleading indentation", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2016-04-26T19:58:32", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160426T20:01:26: 
* david.schultz commented:

> In changeset 145088:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="145088"
> fix #1652
> ```


* david.schultz changed resolution from "" to "fixed"
* david.schultz changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:57: 
* olivas changed milestone from "IceRec V5" to ""
* olivas commented:

> Milestone IceRec V5 deleted
